docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -6,3 +6,10 @@ This guide contains instructions to upgrade from version v7.0.0-beta6 to Unrelea
 There you can find links to upgrade notes for other versions too.
 
 [Upgrade from v7.0.0-beta6 to Unreleased]: https://github.com/shopsys/shopsys/compare/v7.0.0-beta6...HEAD
+
+## [shopsys/framework]
+### Application
+- to add support of functional tests of Redis, download [snc_redis.yml](https://github.com/shopsys/project-base/blob/master/app/config/packages/test/snc_redis.yml) to `app/config/packages/test/snc_redis.yml`
```

Comment:
I think there should be instruction about the added test. mayby low priority with link where you can get inspired